ocpp-1-6-api-adapter/src/test/kotlin/com/izivia/ocpp/adapter16/test/MapperTest.kt

```diff
@@ -513,10 +519,17 @@ class MapperTest {
                     stackLevel = 2,
                     chargingProfilePurpose = ChargingProfilePurposeType.ChargePointMaxProfile,
                     chargingProfileKind = ChargingProfileKindType.Absolute,
-                    chargingSchedule = ChargingSchedule(ChargingRateUnitType.A, listOf(ChargingSchedulePeriod(1, 1)))
+                    chargingSchedule = ChargingSchedule(ChargingRateUnitType.A, listOf(ChargingSchedulePeriod(1, 1))),
+                    transactionId = 1
                 )
             )
-        )
+        ).let {
+            it.copy(
+                chargingProfile = it.chargingProfile.copy(
+                    transactionId = transactionIds.getLocalIdByTransactionId(1).localId
+                )
+            )
+        }
```

Comment:
if we perform the update/copy here, we don't test the transaction mapping itself. We need to have a test that doesn't pass without the code change, and passes only after the change. Here I'm pretty sure it passes even if you remove the code adding the transaction id mapping. I think the test is not at the right level, it should be a test of the  `Ocpp16CSApiAdapter` in which you have made the transaction mapping.